app/javascript/alchemy_admin.js

```diff
@@ -16,7 +16,6 @@ import Initializer from "alchemy_admin/initializer"
 import { LinkDialog } from "alchemy_admin/link_dialog"
 import pleaseWaitOverlay from "alchemy_admin/please_wait_overlay"
 import Spinner from "alchemy_admin/spinner"
-import PagePublicationFields from "alchemy_admin/page_publication_fields"
```

Comment:
Is this strictly related to the the sitemap rendering? If not it can be its own PR. I like it though!